projects/1.18/assets/fuzes-relics/fuze_relics/lang/zh_cn.json

```
@@ -0,0 +1,152 @@
+  "item.fuze_relics.cannon_projectile": "加农炮弹药",
+  "item.fuze_relics.cannon_spawn_egg": "加农炮刷怪蛋",
+  "item.fuze_relics.capacity_bottle_chicken": "能力瓶",
+  "item.fuze_relics.capacity_bottle_cow": "能力瓶",
+  "item.fuze_relics.capacity_bottle_creeper": "能力瓶",
+  "item.fuze_relics.capacity_bottle_enderman": "能力瓶",
+  "item.fuze_relics.capacity_bottle_fire": "能力瓶",
+  "item.fuze_relics.capacity_bottle_ocelot": "能力瓶",
+  "item.fuze_relics.capacity_bottle_pinguin": "能力瓶",
+  "item.fuze_relics.capacity_bottle_slime": "能力瓶",
+  "item.fuze_relics.capacity_bottle_water": "能力瓶",
+  "item.fuze_relics.capacity_bottle_zombie": "能力瓶",
+  "item.fuze_relics.care_package_spawn_egg": "空投包刷怪蛋",
+  "item.fuze_relics.coffee_beans": "咖啡豆",
+  "item.fuze_relics.coffee_cup_render_spawn_egg": "咖啡机渲染实体刷怪蛋",
+  "item.fuze_relics.coffee_machine_render_spawn_egg": "咖啡机渲染实体刷怪蛋",
+  "item.fuze_relics.crocodile_bag": "§e鳄鱼皮手提包",
+  "item.fuze_relics.crocodile_egg": "§e鳄鱼蛋",
+  "item.fuze_relics.crocodile_scales": "§e鳄鱼鳞片",
+  "item.fuze_relics.crocodile_spawn_egg": "鳄鱼刷怪蛋",
+  "item.fuze_relics.cup_of_coffee": "一杯咖啡",
+  "item.fuze_relics.didier_cannon_spawn_egg": "迪迪埃加农炮刷怪蛋",
+  "item.fuze_relics.didier_lucky_block_spawn_egg": "幸运迪迪埃刷怪蛋",
+  "item.fuze_relics.didier_spawn_egg": "迪迪埃刷怪蛋",
+  "item.fuze_relics.drone_item": "无人机",
+  "item.fuze_relics.drone_spawn_egg": "无人机刷怪蛋",
+  "item.fuze_relics.ender_bee_spawn_egg": "末影蜂刷怪蛋",
+  "item.fuze_relics.flashlight": "手电筒",
+  "item.fuze_relics.fuze_guitare": "Fuze 吉他",
+  "item.fuze_relics.fuze_reine_des_neiges": "§b音乐唱片",
+  "item.fuze_relics.fuze_reine_des_neiges.desc": "Fuze - fuze reine des neiges",
+  "item.fuze_relics.fuze_relics_icon": "Fuze Relics Icon",
+  "item.fuze_relics.giant_squid_spawn_egg": "巨型鱿鱼刷怪蛋",
+  "item.fuze_relics.grapplin_hook": "抓钩",
+  "item.fuze_relics.le_clash_des_anti_fuze": "§b音乐唱片",
+  "item.fuze_relics.le_clash_des_anti_fuze.desc": "Fuze - le clash des anti-fuze",
+  "item.fuze_relics.mucus_bucket": "黏液桶",
+  "item.fuze_relics.nether_portal_entity_spawn_egg": "下界传送门实体刷怪蛋",
+  "item.fuze_relics.nether_portal_gun": "§d下界传送枪",
+  "item.fuze_relics.on_m_appelle_fuzeiii": "§b音乐唱片",
+  "item.fuze_relics.on_m_appelle_fuzeiii.desc": "Fuze - on m'appelle fuzeIII",
+  "item.fuze_relics.paladium_go": "§b音乐唱片",
+  "item.fuze_relics.paladium_go.desc": "Fuze - paladium go",
+  "item.fuze_relics.pinguin_hide": "企鹅皮",
+  "item.fuze_relics.pinguin_spawn_egg": "企鹅刷怪蛋",
+  "item.fuze_relics.piston_body": "活塞底座",
+  "item.fuze_relics.piston_hand": "活塞头",
+  "item.fuze_relics.pulsion_boots": "脉冲靴子",
+  "item.fuze_relics.pulsion_chestplate": "§b脉冲胸甲",
+  "item.fuze_relics.pulsion_helmet": "脉冲头盔",
+  "item.fuze_relics.pulsion_leggings": "脉冲护腿",
+  "item.fuze_relics.re_fuzay": "§b音乐唱片",
+  "item.fuze_relics.re_fuzay.desc": "Furti_Two - refuzay",
+  "item.fuze_relics.remote_care_package": "空投呼叫器",
+  "item.fuze_relics.remote_controler": "无人机遥控器",
+  "item.fuze_relics.slide_boots": "滑靴",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "item.fuze_relics.slide_boots": "滑雪靴",
```
和下方的胸甲等一致
原文：**Slide Boots**